src/gpodder/plugins/soundcloud.py

```diff
@@ -90,11 +92,29 @@ def get_user_info(self):
         global CONSUMER_KEY
         key = ':'.join((self.username, 'user_info'))
         if key in self.cache:
-            return self.cache[key]
+            if self.cache[key].get('code', 200) == 200:
+                return self.cache[key]
 
         try:
-            json_url = 'https://api.soundcloud.com/users/%s.json?consumer_key=%s' % (self.username, CONSUMER_KEY)
-            user_info = util.urlopen(json_url).json()
+            # find user ID in soundcloud page
+            url = 'https://soundcloud.com/' + self.username
+            r = requests.get(url)
```

Comment:
why not use **util.urlopen**?

Reply:
Oops! I read it as *urllib* and blindly replaced it with requests. I'll fix that and change the youtube.py code to also use it.